packages/utils/test/schema/getDefaultFormStateTest.ts

```diff
@@ -412,6 +412,226 @@ export default function getDefaultFormStateTest(testValidator: TestValidatorType
+            requiredArray: {
+              type: 'array',
+              items: { type: 'string' },
+              minItems: 1,
+            },
+          },
+          required: ['requiredArray'],
+        };
+
+        expect(
+          computeDefaults(testValidator, schema, {
+            rootSchema: schema,
+            experimental_defaultFormStateBehavior: { arrayMinItems: { populate: 'never' } },
+          })
+        ).toStrictEqual({ requiredArray: [] });
+      });
+      it('should not be filled if minItems defined and non required', () => {
+        const schema: RJSFSchema = {
+          type: 'object',
+          properties: {
+            nonRequiredArray: {
+              type: 'array',
+              items: { type: 'string' },
+              minItems: 1,
+            },
+          },
+        };
+        expect(
+          computeDefaults(testValidator, schema, {
+            rootSchema: schema,
+            experimental_defaultFormStateBehavior: { arrayMinItems: { populate: 'never' } },
+          })
+        ).toStrictEqual({ nonRequiredArray: [] });
+      });
+
+      it('should be filled with default values if required', () => {
+        const schema: RJSFSchema = {
+          type: 'object',
+          properties: {
+            requiredArray: {
+              type: 'array',
+              default: ['raw0'],
+              items: { type: 'string' },
+              minItems: 1,
+            },
+          },
+          required: ['requiredArray'],
+        };
+        expect(
+          computeDefaults(testValidator, schema, {
+            rootSchema: schema,
+            experimental_defaultFormStateBehavior: { arrayMinItems: { populate: 'never' } },
+          })
+        ).toStrictEqual({ requiredArray: ['raw0'] });
+      });
+
+      it('should be filled with default values if non required', () => {
+        const schema: RJSFSchema = {
+          type: 'object',
+          properties: {
+            nonRequiredArray: {
+              type: 'array',
+              default: ['raw0'],
+              items: { type: 'string' },
+              minItems: 1,
+            },
+          },
+        };
+        expect(
+          computeDefaults(testValidator, schema, {
+            rootSchema: schema,
+            experimental_defaultFormStateBehavior: { arrayMinItems: { populate: 'never' } },
+          })
+        ).toStrictEqual({ nonRequiredArray: ['raw0'] });
+      });
+
+      it('should be filled with default values partly and not fill others', () => {
+        const schema: RJSFSchema = {
+          type: 'object',
+          properties: {
+            nonRequiredArray: {
+              type: 'array',
+              default: ['raw0'],
+              items: { type: 'string' },
+              minItems: 2,
+            },
+          },
+        };
+        expect(
+          computeDefaults(testValidator, schema, {
+            rootSchema: schema,
+            rawFormData: { nonRequiredArray: ['raw0'] },
```

Comment:
Might I suggest a different value than the default so that we can see which value is actually being chosen?

Reply:
Of course. Which one?

Reply:
```suggestion
            rawFormData: { nonRequiredArray: ['raw1'] },
```

Reply:
done